src/quarot/gptq.py

```diff
@@ -213,7 +221,9 @@ def quantize_model_gptq(
     apply_mask: bool = False,
 ) -> None:
     """
-    TODO
+    Quantize the model in-place using the GPTQ scheme, using the dataloader calibration data. All weights are stored in FP16.
+    If the model is a QuaRot model, the weights-minus-offsets and scales are stored in the QuarotFP16Linear modules. If the model is not a QuaRot model,
```

Comment:
wouldn't it be cleaner for the QuaRotLinear module to contain an optional offset?